v3/lints/rfc/lint_serial_number_longer_than_20_octets_test.go

```diff
@@ -38,3 +38,12 @@ func TestSnNotTooLarge(t *testing.T) {
 		t.Errorf("%s: expected %s, got %s", inputPath, expected, out.Status)
 	}
 }
+
+func TestSnTooLargeDueToSignedMSB(t *testing.T) {
```

Comment:
Not strictly necessary but it would be nice if you could rework the three separate test functions in a table-driven style. We generally prefer those for new tests.

Reply:
Improvements are always welcome!